custom_components/tesla_custom/util.py

```diff
@@ -9,3 +9,19 @@
     from homeassistant.util.ssl import client_context
 
     SSL_CONTEXT = client_context()
+
+
+def km_to_miles(odometer: float) -> float:
```

Comment:
We shouldn't have any conversion code. Is there no way to use HA native conversion code? Normally you just declare the native sensor units and HA figures it out.